xml/book_gnomeuser.xml

```diff
@@ -33,6 +33,24 @@
    </dm:bugtracker>
    <dm:translation>yes</dm:translation>
   </dm:docmanager>
+  <meta name="title" its:translate="yes">&gnomeuser;</meta>
+  <meta name="series" its:translate="no">Products &amp; Solutions</meta>
+  <meta name="description" its:translate="yes">How to use the GNOME desktop on &productname;</meta>
+  <meta name="social-descr" its:translate="yes">Use the GNOME desktop</meta>
+  <meta name="task" its:translate="yes">
```

Comment:
@janajaeger : For the GNOME Smart Doc, I noticed that @jfaltenbacher used different tasks (which I find a bit more appropriate): 

<meta name="task">
        <phrase>Configuration</phrase>
        <phrase>Installation</phrase>
</meta>

Should we agree on using the latter? If yes, I need to update this in all the other maintenance branches, too.

Reply:
If that means that you have to touch it in four or five other branches, I'd say, don't. We have far too much metadata on our hands as it is. Rather revert Julia's to be consistent?

Reply:
from my point of view: The Gnome User guide is way more extensive and not only features first steps but also maintenance instructions whereas the Smart Docs article only focuses on first steps (Installation). But Tanja and me can discuss tomorrow in our call

Reply:
That difference makes sense, Julia - then I suggest to leave the tasks as they are for the GNOME Smart Doc and the GNOME User Guide.

Reply:
@jfaltenbacher make sure your task meta tag also carries the its:translate="no" attribute ... not relevant to this PR, but as we a are discussing the metadata here, I thought, I'd raise it here ;)